src/Command/Pull/PullCommandBase.php

```diff
@@ -916,11 +921,36 @@ protected function getDatabaseBackup(
     }
     $backup_response = $backups_response[0];
     $this->logger->debug('Using database backup (id #' . $backup_response->id . ') generated at ' . $backup_response->completedAt);
+
+    return $backup_response;
+  }
+
+  /**
+   * Print information to the console about the selected database backup.
+   *
+   * @param bool $backup_response
+   * @param \AcquiaCloudApi\Response\EnvironmentResponse $source_environment
+   */
+  protected function printDatabaseBackupInfo(
+    bool $backup_response,
+    EnvironmentResponse $source_environment
+  ): void {
     $interval = time() - strtotime($backup_response->completedAt);
-    if ($interval > 24 * 60 * 60) {
```

Comment:
@danepowell This is a bug. Should be dividing.

Reply:
I don't see any bug here. `$interval` is seconds, `24 * 60 * 60` is the number of seconds in a day, so this is the correct way to determine if the backup is more than 24 hours (24*60*60 seconds) old.

Anyway, if you'd like to print the actual age that's fine too.